src/common/options.cc

```diff
@@ -4530,11 +4535,11 @@ std::vector<Option> get_global_options() {
     .set_description("Rocksdb options"),
 
     Option("bluestore_rocksdb_cf", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
-    .set_default(false)
+    .set_default(true)
     .set_description("Enable use of rocksdb column families for bluestore metadata"),
 
     Option("bluestore_rocksdb_cfs", Option::TYPE_STR, Option::LEVEL_DEV)
-    .set_default("M= P= L=")
+    .set_default("O=")
```

Comment:
@aclamk would this have any effect on your sharding patches?

Reply:
@jdurgin Yes, this PR will affect my sharding. Once this is merged, I will need to adapt to it, but its impact is very limited.